testing/zsh-syntax-highlighting/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Dawid Dziurla <dawidd0811@gmail.com>
+# Maintainer: Dawid Dziurla <dawidd0811@gmail.com>
+pkgname=zsh-syntax-highlighting
+pkgver=0.5.0
+pkgrel=0
+pkgdesc="Fish shell like syntax highlighting for Zsh."
+url="https://github.com/zsh-users/zsh-syntax-highlighting"
+arch="noarch"
+license="BSD-3"
+depends="zsh"
+makedepends=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::$url/archive/$pkgver.tar.gz"
+options="!check"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+package() {
+	cd "$builddir"
+	make install DESTDIR="$pkgdir" PREFIX=/usr
```

Comment:
Wouldn’t you mind changing destination directory to `/usr/share/zsh/plugins/$pkgname`? It’s IMO better to install all ZSH plugins into common directory, for ease of use and possible integration with ZSH plugin systems.

```sh
make install DESTDIR="$pkgdir" PREFIX="/usr" SHARE_DIR="$pkgdir/usr/share/zsh/plugins/$pkgname"
```